fr3d/cif/reader.py

```diff
@@ -299,32 +309,36 @@ def operator(entry):
     def __atom__(self, pdb, atom, symmetry):
         x, y, z = self.__apply_symmetry__(atom, symmetry)
 
-        index = atom['label_seq_id']
+        index = atom['label_seq_id'] if 'label_seq_id' in atom else '.'
```

Comment:
It would be a bit more straighforward to just use:

```python
index = atom.get('label_seq_id', '.')
```
if possible. This is a general change for the code, but I'm just noting it here.